pkg/apis/workflow/v1alpha1/cron_workflow_types.go

```diff
@@ -69,6 +78,12 @@ type CronWorkflowStatus struct {
 	LastScheduledTime *metav1.Time `json:"lastScheduledTime" protobuf:"bytes,2,opt,name=lastScheduledTime"`
 	// Conditions is a list of conditions the CronWorkflow may have
 	Conditions Conditions `json:"conditions" protobuf:"bytes,3,rep,name=conditions"`
+	// Succeeded is a counter of how many times the child workflows had success
+	Succeeded int64 `json:"succeeded" protobuf:"varint,4,rep,name=succeeded"`
+	// Failed is a counter of how many times a child workflow terminated in failed or errored state
+	Failed int64 `json:"failed" protobuf:"varint,5,rep,name=failed"`
+	// Completed is a flag that is set to true when the stopping condition is achieved which stops new CronWorkflows from running
+	Completed bool `json:"completed" protobuf:"varint,6,rep,name=completed"`
```

Comment:
That makes sense. Maybe `active` instead of `running`?

Reply:
When would the phase be failed? If `checkStoppingCondition` fails to evaluate? Would the cron continue to fire if that happens?

Reply:
IMO the cron shouldn't fire on an invalid `condition`

Reply:
I'm still not sure when we would enter `phase: failed` then - I'm just trying to determine what these three phases would mean.

Reply:
Let's wait for @sarabala1979  input  but I understood it as:
`failure` - if a condition using the `failed` keyword is achieved
`complete` - if any other type of condition is achieved 

However, this increases complexity because we need to check the stopping condition and which type of condition (success/fail) was achieved. 

If you think that it is better to have a phase, for now, I would prefer to have it with `active` or `terminated`.
It could also have a `error` phase to handle the case where `checkStoppingCondition` fails to evaluate, however, IMO it makes more sense to just throw a spec error.

Reply:
I agree with all you've written here. I had assumed we'd want to support `succeeded+failed=5` as a condition, which isn't either failure type, hence the confusion.

I also agree with you on how to handle `checkStoppingCondition` failure to evaluate.

Reply:
@eduardodbr Sorry for the long delay for reply. my comment was for only cron workflow `phase`, as per PR cronworkflow is running or not is based on `completed`
instead of `boolean` can we have `phase` with `active` and `stopped` ?

Reply:
``` yaml
   phase: active/stopped
```